scripts/ruby/lib/dataManager.rb

```diff
@@ -219,6 +198,24 @@ def prepare_files
     return file_classes
   end
 
+  def set_schema
+    # if ES is requested and not transform only, then set the schema
+    # to make sure that any new fields are stored with the correct fieldtype
+    if should_transform?("es") && !@options["transform_only"]
+      schema = YAML.load_file("#{@repo_dir}/#{@options["es_schema_path"]}")
+      path, idx, type = ["es_path", "es_index", "es_type"].map { |i| @options[i] }
+      url = "#{path}/#{idx}/_mapping/#{type}?pretty&update_all_types"
+      begin
+        RestClient.put(url, schema.to_json, { content_type: :json })
+        msg = "Successfully set elasticsearch schema for #{type}"
+        @log.info(msg)
```

Comment:
The PR looks great, but I learned in my research about Rails logging, that it's best to pass blocks to logging methods. http://edgeguides.rubyonrails.org/debugging_rails_applications.html#impact-of-logs-on-performance

I wrote my logging calls in my other PR that way and plan to going forward. Shall I create a separate issue to update the code for that in this repo?

Reply:
Sounds good to me!